common/datavalues/src/types/data_type.rs

```diff
@@ -91,6 +91,11 @@ pub trait DataType: std::fmt::Debug + Sync + Send + DynClone {
 
     fn name(&self) -> &str;
 
+    /// Returns the name to display in the SQL describe
+    fn sql_name(&self) -> String {
+        self.name().to_lowercase()
```

Comment:
to_uppercase?

Reply:
See the summary of this pr.

Reply:
I think this is a bad case for MySQL, his style is inconsistent.
Most databases is using uppercase, e.g. snowflake, duckdb

Our document changed to uppercase for the datatype now, realize that one of the benefits of uppercase is:

With uppercase:
```
CREATE TABLE t1(int INT);
```

With lowercase:
```
CREATE TABLE t1(int int); -- int is a field name or data type is trivial here
```

Reply:
ok